db/alembic/versions/08eca0bda924_add_machine_users_and_api_keys.py

```diff
@@ -0,0 +1,82 @@
+"""add_machine_users_and_api_keys
+
+Revision ID: 08eca0bda924
+Revises: 831ad1e71a80
+Create Date: 2025-09-01 09:06:34.832320
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "08eca0bda924"
+down_revision: Union[str, None] = "831ad1e71a80"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    """Upgrade schema."""
+    # Add machine user fields to users table
+    op.add_column(
+        "users", sa.Column("is_machine_user", sa.Boolean(), default=False)
```

Comment:
@sunu can you explain why we need to have this boolean field in addition to having a separate MACHINE_USER type?

Reply:
@batpad thanks for the catch! I don't think we need a separate field. I've removed it.